openmc/surface.py

```diff
@@ -1880,6 +1880,26 @@ def bounding_box(self):
     def __str__(self):
         return '-' + str(self.surface.id) if self.side == '-' \
             else str(self.surface.id)
+    
+    def get_surfaces_from_region(self, surfaces = OrderedDict()):
+        """
+        Returns the surface that this is a halfspace of.
+        Overwrites method Region.get_surfaces_from_region()
+    
+        Parameters
+        ----------
+        surfaces: collections.OrderedDict, optional
+            Dictionary mapping surface IDs to :class:`openmc.Surface` instances
+    
+        Returns
+        -------
+        surfaces: collections.OrderedDict
+            Dictionary mapping surface IDs to :class:`openmc.Surface` instances
+    
+        """
+        if self.surface.id not in surfaces:
```

Comment:
This is fine, though my personal preference would be for the more succinct but equally functional:

```python
surfaces[self.surface.id] = self.surface
return surfaces
````